docs/monthly/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta
+      http-equiv="Refresh"
```

Comment:
```suggestion
      http-equiv="refresh"
```

Reply:
Damn.. didn't want to commit this as it didn't work :D